js/app/generator.js

```diff
@@ -174,12 +178,24 @@ function(config, Quad, grid){
     }
 
     /**
+     * Set the current level.
+     */
+    Generator.prototype.setLevel = function(level) {
+        this.level = level;
+
+        var speed = config.speeds[this.level];
+        this.dropTimer.loop(speed * Phaser.Timer.SECOND, this.drop.bind(this));
+    }
+
+    /**
      * Draw the graphics showing the time remaining before the next drop
      */
     Generator.prototype.drawTimerGraphics = function() {
+        var speed = config.speeds[this.level];
+
         this.timerGraphic.clear();
         var percentElapsed =
-                this.dropTimer.ms/(config.generator.defaultWait*1000)
+                this.dropTimer.ms/(speed * 1000)
```

Comment:
Please use the Phaser.Time.SECOND constant here as well (I should have originally)